game.py

```diff
@@ -12,16 +12,41 @@
     '-----------'
 ]
 
-
 def snowman(snowman_word):
     """Complete the snowman function
     replace "pass" below with your own code
     It should print 'Congratulations, you win!'
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+   
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
```

Comment:
Nice job using the built-in function `len` to keep track of how many incorrect guesses there are in the game!